config/default.yml

```diff
@@ -501,6 +501,15 @@ Rails/SkipsModelValidations:
     - update_counters
   Whitelist: []
 
+Rails/TagInsteadOfContentTag:
+  Description: 'Use `tag` instead of `content_tag`.'
+  Reference:
+    - 'https://github.com/rails/rails/issues/25195'
+    - 'https://api.rubyonrails.org/classes/ActionView/Helpers/TagHelper.html#method-i-content_tag'
+  Enabled: false
+  Safe: false
```

Comment:
Why is it not safe?

Reply:
Thank you for your review!

I think here are two reasons.

1. When first argument of content_tag is not symbol or string, the result may change from broken html to syntax error.

```ruby
[1] pry(main)> cd helper
[2] pry(#<ActionView::Base>):1> content_tag({foo: 1})
=> "<{:foo=>1}></{:foo=>1}>"
[3] pry(#<ActionView::Base>):1> tag.{foo: 1}
SyntaxError: unexpected {
tag.{foo: 1}
```

2. Without arguments, the result changes.

```ruby
[1] pry(main)> cd helper
[2] pry(#<ActionView::Base>):1> tag.br
=> "<br>"
[3] pry(#<ActionView::Base>):1> content_tag(:br)
=> "<br></br>"
```

Reply:
> 1. when first argument of content_tag is not symbol or string

Please add parentheses with auto-correction when the argument is a hash.

```ruby
content_tag({foo: 1}) #=> "<{:foo=>1}></{:foo=>1}>"
tag({foo: 1})         #=> "<{:foo=>1} />"
```

> 2. Without arguments, the result changes.

I think that it is an acceptable change.

Reply:
I found that `content_tag("foo-bar")` is unsafe too.
I fixed these cases.

And I fixed all problems you reviewed!